### ISSUE TITLE
Checkboxes are hard to read on mobile for long options

### ISSUE THREAD

**Issue by pontaoski** [cryptpad/cryptpad#1507]
### Description

![grafik](https://github.com/cryptpad/cryptpad/assets/20326855/66f2b796-f7e1-4e17-8047-a09d2ac03319)

People responding to my forms have commented that they can't really tell which checkboxes go with which text on mobile.

### Steps to reproduce

1. Create a form with a checkbox choice
2. Create decently wordy options
3. View the form

### Expected behavior

The options should be easily distinguishable from each other

### Which operating system are you using?

Linux/BSD/UNIX, Microsoft Windows, Apple iOS, Google Android

### What browsers are you seeing the problem on?

Mozilla Firefox, Google Chrome, Apple Safari, Microsoft Edge

### Extensions

reproducible with none

### Version

2024.3.1
